scripts/commands/wizard/quiz.js

```diff
@@ -0,0 +1,175 @@
+const api = require('../../../src/api')
+
+const {
+  selectTutorial,
+  promptFilesReady,
+  selectLesson,
+  logList,
+  validateStringPresent,
+  promptRepeat,
+  logPreview,
+  logCreateLater
+} = require('./utils.js')
+
+// *** QUIZ CREATION ***
+
+async function createQuiz (tutorial, lesson, { createLesson, createTutorial, createResource }) {
+  log.info("Now it's time to write the question for your multiple-choice quiz and provide answer choices, with positive or negative feedback for each. Wrong answers, and the feedback associated with them, are a great way to address common misconceptions about the topic. Be sure to make your feedback as helpful as possible, to guide the learner to the right choice. You'll need to create 1 correct answer and 2-3 incorrect answers. (I'll take care of randomizing their order later.)")
+
+  let responses = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'question',
+      message: 'What question should the learner answer?',
+      validate: validateStringPresent
+    },
+    {
+      type: 'input',
+      name: 'correctAnswer',
+      message: "What's the correct answer to this question?",
+      validate: validateStringPresent
+    },
+    {
+      type: 'input',
+      name: 'correctFeedback',
+      message: 'What success message should we show the learner if they pick this option?',
+      validate: validateStringPresent
+    }
+  ])
+
+  const question = responses.question
+
+  const choices = [
+    {
+      answer: responses.correctAnswer,
+      correct: true,
+      feedback: responses.correctFeedback
+    }
+  ]
+  log.info(`Question: "${question}"`)
+  log.info(`Answer (Correct): "${choices[0].answer}"`)
+  log.info(`Feedback: "${choices[0].feedback}"`)
+
+  let askAgain = true
+  let wrongAnswer
+
+  while (askAgain) {
+    // repeat as many times as they want to create new wrong answers
+    responses = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'incorrectAnswer',
+        message: 'What incorrect option should be presented to the learner?',
+        validate: validateStringPresent
+      },
+      {
+        type: 'input',
+        name: 'incorrectFeedback',
+        message: 'What failure message should the learner see if they pick this option? Be sure to explain why this choice is wrong so they can feel more confident when choosing again.',
+        validate: validateStringPresent
+      }
+    ])
+
+    wrongAnswer = {
+      answer: responses.incorrectAnswer,
+      correct: false,
+      feedback: responses.incorrectFeedback
+    }
+
+    log.info(`Answer (Incorrect): "${wrongAnswer.answer}"`)
+    log.info(`Feedback: "${wrongAnswer.feedback}"`)
+
+    choices.push(wrongAnswer)
+
+    log.info(`You currently have 1 correct answer and ${choices.length - 1} wrong answer${choices.length > 2 ? 's' : ''}. (I recommend providing 2-3 incorrect options.)`)
+
+    if (!(await promptRepeat('wrong answer'))) {
+      askAgain = false
+    }
```

Comment:
Just a suggestion to simplify the logic:

```suggestion
    askAgain = await promptRepeat('wrong answer') // prompt to repeat adding a wrong answer
```